### ISSUE TITLE
Sync user notes and block list

### ISSUE THREAD

**Issue by matrixik** [insin/hn-comments-owl#26]
Hello, thank you for working on this addon.

Do you plan adding synchronisation support?

I read HN on two computers and two phones, Firefox (with sync always on) everywhere and it would be great if user notes and block list would be synchronized between them.

**Comment by insin** [comment]
Yes, will add this, not having it is a hangover from the extension's userscript origins - if we switch all storage to use `chrome.storage.sync` and abstract that so the userscript version falls back to `localStorage`, we should get this pretty much for free.
